src/providers/AprsProvider.tsx

```diff
@@ -15,13 +15,18 @@ type MinAndMax = {
   max: BigNumber
 }
 type AmountReward = {
-  amount: MinAndMax
+  rewardToken: BasicToken
+  amountPerDay: MinAndMax
+  apr?: undefined
```

Comment:
Should this be `number` or `string`?

Reply:
nope these are complementary types - one defines what the other leaves undefined. It allows us to both be unique and do `if (reward.apr) {...} else { reward.amount.min... }` while also having an overlapping type `RewardAmount | RewardApr`